### PR TITLE
74 map selection

### PR DESCRIPTION
**Pull request by Klemensas**
Closes https://github.com/spry-group/trashradar-api/issues/74

### CODE REVIEW

**Comment by marton987** [comment]
Great job @Klemensas looks very good

Can we disable street view and the option to select the point to avoid something like:
screen shot 2017-06-16 at 10 31 18 pm
screen shot 2017-06-16 at 10 31 44 pm
screen shot 2017-06-16 at 10 32 34 pm

**Comment by Klemensas** [comment]
I've disabled the street view and clickable icons.

File: src/app/main/complaint/complaint.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      declarations: [ ComplaintComponent ],
+      schemas: [ CUSTOM_ELEMENTS_SCHEMA ],
+      imports: [
+        FormsModule,
+        AgmCoreModule.forRoot({
+          apiKey: environment.googleMapsAPIKey,
+          libraries: ['places']
+        })
+      ],
+      // providers: [{
```
Comment (marton987) [inline src/app/main/complaint/complaint.component.spec.ts]: Can we remove this?

**Comment by marton987** [comment]
LGTM great job

**Pull request merged**